packages/odo-expandable/src/expandable-group.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @fileoverview A wrapper for multiple Expandable elements that will
+ * allow them to operate coherently in an accordion type fashion.
+ *
+ * @author Matt Zaso
+ */
+import Settings from './settings';
+import ExpandableItem from './expandable-item';
+
+class ExpandableGroup {
+  constructor(elements) {
+    /** @type {Array.<!Element>} */
+    this._elements = elements;
+
+    /** @type {Array.<!ExpandableItem>} */
+    this._expandables = elements.map(trigger => new ExpandableItem(
```

Comment:
Let's make this `@protected`. For Odo components, that just means no leading underscore and add the `@protected` jsdoc tag. Then change the subclass to use the new name.